M:tG Life.xcodeproj/project.pbxproj

```diff
@@ -46,14 +52,25 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		BA1B8F051C3E16E700027FB0 /* AllCards.json */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; path = AllCards.json; sourceTree = "<group>"; };
```

Comment:
Pretty sure you left this file uncommitted (correctly), but it's still in the project. I'll find a better solution than needing it in the bundle but could you remove it from the project.